tools/releasing/create_binary_release.sh

```diff
@@ -68,7 +68,11 @@ make_binary_release() {
   $MVN clean package -Prelease -am -Dgpg.skip -Dcheckstyle.skip=true -DskipTests
 
   cd fluss-dist/target/fluss-${RELEASE_VERSION}-bin
+  cp ${FLUSS_DIR}/LICENSE ./fluss-${RELEASE_VERSION}/LICENSE
```

Comment:
That's the `LICENSE` from the source distribution right ? 

I guess we need a specific one for the binary distribution (as we bundle dependencies).

Maybe we can do another PR specifically for `LICENSE` and `NOTICE` in the binary distribution.